### DISCUSSION TITLE
Tag Viewer (because tumblr's new notes section misses things)

### DISCUSSION THREAD

**Discussion by irideal** [AprilSylph/XKit-Rewritten#339] [Suggestion Box]
Is there a possibility of bringing back some form of TagViewer? 

I know that up until the recent update, tumblr had this feature built in, but with the # button removed, it no longer works properly. Allegedly, tumblr allows you to view reblog tags in the "Reblogs with comment" section of the notes, but I've found that this misses many reblogs. I had thought some posts of mine had more tag comments than were displayed (in some cases it told me there were none), so I went back to those reblogs to check, and sure enough there were tags.

Additionally, although it gives a list of people who reblogged, it links to the homepage of their blog rather than the post, so it was extremely annoying to even find those posts on their blogs to double check if I was misremembering whether things had gotten tag comments.

![tumblr tag problem evidence](https://user-images.githubusercontent.com/95011374/143383906-e93b026d-11a7-43e0-9314-9c83dda6d2ef.png)

**Comment by AprilSylph** [comment]
Please report this to [Tumblr support](https://www.tumblr.com/support) if you haven't already. 

As for an XKit-based solution, this new notes UI is still experimental, so I won't consider any action until that changes.

**Reply by irideal** [reply]
I did submit this to tumblr back in last November, as well as followed up with them twice. All of these times I got a response that they would look into it and post about it on the changes blog, but this has never happened to the best of my knowledge.

The new notes view has been rolled out to everyone for months now, and the problem is still the same as it was back then. I've also noticed that it may be an age related issue, as older posts of mine are much more likely to have missing tagged reblogs, and past a certain point all of the posts claim there are "No reblogs yet with this filter".

It's extremely frustrating since the old note viewing system did not have this issue with not pulling all reblogs. The new notes view would be a better design if it actually worked properly! I don't know how much faith I have in staff ever addressing this issue.
